corelib/corelib.pro

```diff
@@ -79,6 +79,7 @@ SOURCES += crayfish.cpp \
     crayfish_capi.cpp \
     crayfish_dataset_dat.cpp \
     crayfish_grib.cpp \
+    crayfish_netCFD.cpp \
```

Comment:
cdf, not cfd :-)
also for consistency better to use just lowercase for filenames